src/main/java/com/rcv/Tabulator.java

```diff
@@ -326,8 +326,16 @@ private void setWinningThreshold(Map<String, BigDecimal> currentRoundCandidateTo
 
     // divisor for threshold is num winners + 1
     BigDecimal divisor = new BigDecimal(config.getNumberOfWinners() + 1);
-    // threshold = floor(votes / (num_winners + 1)) + 1
-    winningThreshold = currentRoundTotalVotes.divideToIntegralValue(divisor).add(BigDecimal.ONE);
+    if (config.isWinningThresholdIntegral()) {
+      // threshold = floor(votes / (num_winners + 1)) + 1
+      winningThreshold = currentRoundTotalVotes.divideToIntegralValue(divisor).add(BigDecimal.ONE);
+    } else {
+      // threshold = (votes / (num_winner + 1)) + 10^(-1 * decimalPlacesForVoteArithmetic)
+      BigDecimal augend =
```

Comment:
Not sure what `augend` means. Can we use a better variable name?

Reply:
It's the formal name for the second argument to an addition operation (and it's what BigDecimal calls the argument to the `add` method). I'm fine calling it something else, but I'm not sure what. `fudgeFactor`?